modules/pool/pool.gql

```diff
@@ -1482,6 +1482,14 @@ input GqlPoolEventsFilter {
     typeIn: [GqlPoolEventType]
     userAddress: String
     range: GqlPoolEventsDataRange
+    """
```

Comment:
while your on it, add comments to all other fields as well

Reply:
the other fields seem self-explanatory enough by the attribute names alone

Reply:
Yeah go a bit further and check other types in this file. We should start adding these with every pr